### ISSUE TITLE
PHP7 Throwable, how to test ?

### ISSUE THREAD

**Issue by Pierozi** [atoum/atoum#592]
Hello,

I've see the `Throwable` exception has been implemented by #537 #536 
but i'm asking what's the recommand way for catch it into test.

This is the behavior I expected

```php
        $this
            ->exception(function() {
                $this->newTestedInstance();
            })
            ->isInstanceOf(\TypeError::class)
        ;
```

But like it's not work, i've workarround like this.

```php

        $this
            ->exception(function() {
                try {
                    new \Plab\GlsUniboxDelivery\Gls\Provider\Config();
                } catch (\TypeError $e) {
                    throw new \Exception('Error');
                }
            })
            ->isInstanceOf(\Exception::class)
        ;
```

**Comment by jubianchi** [comment]
Hi @Pierozi !

Your first snippet is totally correct but it does not work because of a bug in atoum. It's now addressed by #593.

**Comment by Pierozi** [comment]
Hello @jubianchi !

thanks, i didn't expect to find an issue. thanks for PR i will update my code when PR will be merged.
